### ISSUE TITLE
AttributeError: 'Config' object has no attribute 'postprocessing/savePostProcessing'

### ISSUE THREAD

**Issue by ripper1967** [pymedusa/SickRage#1017]
### Before submitting your issue:

Enable debug logging in SickRage settings, reproduce the error (be sure to disable after the bug is fixed)

Branch/Commit:
OS:
What you did:Tried to save a setting
What happened: nothing errors out
What you expected:to save in the config
Logs:
```
2016-09-01 08:15:09 DEBUG    TORNADO :: [004d0d0] Failed doing web ui request u'postprocessing/savePostProcessing': Traceback (most recent call last):
  File "C:\pymedusaSickRage\sickbeard\server\web\core\base.py", line 252, in get
    method = getattr(self, route)
AttributeError: 'Config' object has no attribute 'postprocessing/savePostProcessing'
```

**Comment by fernandog** [comment]
@xo @ratoaq2 any idea whats wrong?

**Comment by themixer2000** [comment]
found the issue to be in /SickRage/gui/slick/views/config_postProcessing.makoconfig_postProccessing.mako  the problem is on line 23             <form id="configForm" action="config/postprocessing/savePostProcessing" method="post"> needs to be changed to this             <form id="configForm" action="config/postProcessing/savePostProcessing" method="post"> just the capital p is missing hope that helps

**Comment by medariox** [comment]
Fixed pymedusa/SickRage#1017

Thanks @themixer2000
